scripts/init_project.py

```diff
@@ -0,0 +1,192 @@
+                u = gconf[i + 1]
+                if not re.match(r"^\s*url =", u):
+                    u = gconf[i + 2]
+                if not re.match(r"^\s*url =", u):
+                    u = None
+                else:
+                    u = re.sub(r"^\s*url = ", '', u)
+                if u != None and re.match(r"^git@github.com:", u):
+                    u = re.sub(r"^git@github.com:", 'https://github.com/', u)
+                return u
+        except Exception as e:
+            return ""
+
+    def _input_str_to_list(self, input_str: str) -> List[str]:
+        return list(map(lambda s: s.strip(), input_str.split(",")))
+
+    def generate_iproj_json(self) -> str:
+        iproj = {
+            "description": self.description,
+            "version": self.version,
+            "license": self.license,
+            "repository": self.repository,
+            "includePath": self.include_path,
+            "objlib": self.objlib,
+            "curlib": self.curlib,
+            "preUsrLibl": self.pre_usr_libl,
+            "postUsrLibl": self.post_usr_libl,
+            "setIBMiEnvCmd": self.set_IBM_i_env_cmd
+        }
+        return json.dumps(iproj, indent=4)
+
+    def generate_ibmi_json(self) -> str:
+        ibmi = {
+            "version": self.version,
+            "build": {
+                "objlib": self.objlib,
+                "tgtCcsid": self.tgt_ccsid
+            }
+        }
+        return json.dumps(ibmi, indent=4)
+
+    def generate_rules_mk(self) -> str:
+        return '\n'.join(['# Check out the documentation on creating the rules at https://github.com/IBM/ibmi-bob/wiki/Create-Rules.mk',
+                          "SUBDIRS :=",
+                          "",
+                          "TRGs :=",
+                          "DTAs :=",
+                          "SQLs :=",
+                          "BNDDs :=",
+                          "PFs :=",
+                          "LFs :=",
+                          "DSPFs :=",
+                          "PRTFs :=",
+                          "CMDs :=",
+                          "SQLs :=",
+                          "MODULEs :=",
+                          "SRVPGMs :=",
+                          "PGMs :=",
+                          "MENUs :=",
+                          "PNLGRPs :=",
+                          "QMQRYs :=",
+                          "WSCSTs :=",
+                          "MSGs :=",
+                          ])
+
+
+def signal_handler(sig, frame):
+    init_cancelled()
+
+
+def yes(input_str: str):
+    """
+    >>> yes("yes")
+    True
+    >>> yes("y")
+    True
+    >>> yes("Yes")
+    True
+    >>> yes("No")
+    False
+    """
+    return input_str.strip().lower() == "yes" or \
+        input_str.strip().lower() == "y" or \
+        input_str.strip() == ""
+
+
+def init_cancelled():
+    print(colored('\nInit cancelled!', Colors.WARNING))
+    sys.exit(0)
+
+
+def prompt(description, default_vaule) -> str:
```

Comment:
sp value

Reply:
default_vaule should be spelled default_value

Reply:
Thanks